mitmproxy/models/tcp.py

```diff
@@ -53,3 +55,22 @@ def __init__(self, client_conn, server_conn, live=None):
 
     def __repr__(self):
         return "<TCPFlow ({} messages)>".format(len(self.messages))
+
+    def match(self, f):
```

Comment:
Where should the corresponding test for this go? I couldn't find the tests for TCPFlow.